generated/python/test/trace.py

```diff
@@ -28,10 +28,19 @@
 trace = trace_pb2.Trace(project_id=PROJECT_ID, trace_id=id, spans=[span])
 traces = trace_pb2.Traces(traces=[trace])
 
+print 'testing patch'
 api.patch_traces(PROJECT_ID, traces)
+print 'patched'
 
+# wait for patch to propagate
```

Comment:
I'm okay with a comment that says "seems to be long enough to avoid races". Remember that these are ad hoc smoke tests that will become obsolete this quarter as we do the automated unit and smoke testing.